bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml

```diff
@@ -522,5 +522,5 @@ spec:
   maturity: alpha
   provider:
     name: Intel
-  version: 1.1.0
-  replaces: openvino-operator.v1.0.0
+  version: 1.2.0
+  replaces: openvino-operator.v1.1.0
```

Comment:
could this name be consistent with the version?

Reply:
it is consistent. new version will be called 1.2.0 to replace 1.1.0.